views/components/GraphTable/helpers.js

```diff
@@ -0,0 +1,15 @@
+import { TOTAL_LABEL } from '../../../constants'
+import { studyCountsToPercentage } from '../../fe-utils/helpers'
+
+export const graphTableRowDataBySiteUtil = (dataBySite) => {
+  const rowData = []
+  dataBySite.forEach((site) => {
+    const { count, targetTotal } = site.totalsForStudy
+    site.counts[TOTAL_LABEL] = count
+    site.targets[TOTAL_LABEL] = targetTotal
+    site.percentages[TOTAL_LABEL] = studyCountsToPercentage(count, targetTotal)
+    rowData.push(site)
+  })
+
+  return rowData
```

Comment:
```suggestion
  return dataBySite.map((site) => {
    const { count, targetTotal } = site.totalsForStudy
    site.counts[TOTAL_LABEL] = count
    site.targets[TOTAL_LABEL] = targetTotal
    site.percentages[TOTAL_LABEL] = studyCountsToPercentage(count, targetTotal)
    rowData.push(site)
  })
```

Reply:
Thank you 
https://github.com/AMP-SCZ/dpdash/pull/193/commits/325e6635a9f09c0b3ce28c6d2a5ab8bfa1369601